Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -108,7 +108,7 @@ private void AuthenticationStateProvider_AuthenticationStateChanged(Task<Authent
       }
       else
       {
-        task.ContinueWith((t) =>
+        task.ContinueWith(_ =>
```

Comment:
@SimonCropp 
Is there any difference in using `task` within the `ContinueWith` instead of the `Task` passed as parameter to it? Can there be subtle differences so we should use the parameter within the continuation to check for completion/fail etc.?

Reply:
@StefanOssendorf oh good point. yes we should use the parameter in this case. because of the variable closure and the lambda.

i will fix